doc: bluetooth: auto-pts: Add troubleshooting section for Windows (#36660)
zephyrproject-rtos/zephyr: doc/guides/bluetooth/autopts/autopts-win10.rst

   :height: 450
   :width: 600
   :align: center

Troubleshooting
================

- "When running actual hardware test mode, I have only BTP TIMEOUTs."

> This is a problem with connection between autopts client and board. There are many possible causes. Try:

Reviewer: ```suggestion
This is a problem with connection between auto-pts client and board. There are many possible causes. Try:
```

- Clean your auto-pts and zephyr repos with

.. code-block::

    git clean -fdx

> then build and flash tester elf again.

Reviewer: This should come with a warning for those that do not realize what `git clean -fdx` will do

- If you have set up Windows on virtual machine, check if guest extensions are installed properly or change USB compatibility mode in VM settings to USB 2.0.

> - Check, if firewall in not blocking python or socat.exe.

Reviewer: ```suggestion
- Check, if firewall in not blocking python.exe or socat.exe.
```

> - Check if board sends ready event after restart (hex 00 00 80 ff 00 00). Open serial connection to board with PuTTy with proper COM and baud rate. After board reset you should see some strings in console.

Reviewer: ```suggestion
- Check if board sends ready event after restart (hex 00 00 80 ff 00 00). Open serial connection to board with e.g. PuTTy with proper COM and baud rate. After board reset you should see some strings in console.
```
